site/content/en/docs/formats/mvtec.md

```diff
@@ -0,0 +1,141 @@
+---
+title: 'MVTec AD'
+linkTitle: 'MVTec AD'
+description: ''
+---
+
+## Format specification
+
+The MVTec AD format specification is available
+[here](https://link.springer.com/content/pdf/10.1007/s11263-020-01400-4.pdf).
+
+The dataset has annotations for detecting abnormal pixels through binary masks
+and it turns into bounding boxes or abnormal labels for supporting classification,
+detection, and segmentation tasks. The MVTec AD dataset is composed of training data only
+for `good` category without any annotation and testing data for both `good` and multiple
+`defective` categories with masks. The dataset contains total 15 kinds of objects or textures.
+
+Supported tasks / formats:
+- The combined format - `mvtec`
+- Image classification - `mvtec_classification`
+- Object detection - `mvtec_detection`
+- Class and segmentation - `mvtec_segmentation`
```

Comment:
Do we have to rename `Class and segmentation` to `Instance segmentation`?

Reply:
I updated. Thanks.